include/nanoflann.hpp

```diff
@@ -1307,7 +1301,7 @@ class KDTreeBaseClass
         }
         ElementType max_spread = -1;
         cutfeat                = 0;
-        ElementType  min_elem = 0, max_elem = 0;
+        ElementType   min_elem = 0, max_elem = 0;
```

Comment:
Thanks for the PR! 
One doubt: You removed "Dimension" type above, but it seems to be used here?

Reply:
"Dimension" and "Offset" type have not be used in class "PooledAllocator"
it just used in class "KDTreeBaseClass" and Derived classes